frame/nft-fractionalization/src/lib.rs

```diff
@@ -285,5 +290,27 @@ pub mod pallet {
 			T::Assets::burn_from(asset_id, account, amount)?;
 			T::Assets::start_destroy(asset_id, None)
 		}
+
+		/// Set the metadata for the newly created asset.
+		fn do_set_metadata(
+			asset_id: AssetIdOf<T>,
+			depositor: &T::AccountId,
+			pallet_account: &T::AccountId,
+			nft_collection_id: &T::NftCollectionId,
+			nft_id: &T::NftId,
+		) -> DispatchResult {
+			let symbol = "FRAC";
+			let name = format!("Frac {nft_collection_id}-{nft_id}");
```

Comment:
I don't know if you want to pad these ids so that its always the same cost?

Reply:
I think it would be less readable on the UI if we pad them